docs/pages/guides/environment-variables.mdx

```diff
@@ -45,7 +46,15 @@ When you run `npx expo start`, `process.env.EXPO_PUBLIC_API_URL` will be replace
 
 Expo CLI loads **.env** files according to the [standard .env file resolution](https://github.com/bkeepers/dotenv/blob/c6e583a/README.md#what-other-env-files-can-i-use) and then replaces all references in your code to `process.env.EXPO_PUBLIC_[VARNAME]` with the corresponding value set in the **.env** files. Code inside **node_modules** is not affected for security purposes.
 
-Every variable must be referenced as a static property in order for it to be inlined. For example, the expression `process.env.EXPO_PUBLIC_KEY` will be rewritten, but `process.env['EXPO_PUBLIC_KEY']` will not.
+### How to read from environment variables
+
+Once an environment variable is loaded into your app, it remains constant during that app's runtime. This means that the variable is a static property. It does not change unless a different value is set for the variable and the app is reloaded.
+
+- <YesIcon small /> **Every environment variable must be referenced as a static property for it to be
+  inlined.** For example, the expression `process.env.EXPO_PUBLIC_KEY` is valid and will be inlined.
```

Comment:
```suggestion
- <YesIcon small /> **Every environment variable must be statically referenced as a property of `process.env` using JavaScript's dot notation for it to be inlined.** For example, the expression `process.env.EXPO_PUBLIC_KEY` is valid and will be inlined.
```

Reply:
Thanks for updating this and sharing the link, TIL!